iceoryx_examples/icedelivery/iox_publisher.cpp

```diff
@@ -60,11 +60,11 @@ int main()
         double sampleValue3 = ct + 233;
         double sampleValue4 = ct + 377;
 
+        //! [usage1]
         // API Usage #1
```

Comment:
I almost don't dare to say it, but could these two lines be combined ;)

Reply:
Doh!